<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Antique Cars Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Antique Cars Table </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Speed (km/h)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price ($)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Image</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchModelLineEdit">
        <property name="placeholderText">
         <string>Search by Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchSpeedLineEdit">
        <property name="placeholderText">
         <string>Search by Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchPriceLineEdit">
        <property name="placeholderText">
         <string>Search by Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="sortButtonsLayout">
      <item>
       <widget class="QPushButton" name="sortModelButton">
        <property name="text">
         <string>Sort by Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortSpeedButton">
        <property name="text">
         <string>Sort by Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortPriceButton">
        <property name="text">
         <string>Sort by Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add Car</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove Car</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Edit Car</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPhotoButton">
        <property name="text">
         <string>Add Photo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkArraySizeButton">
        <property name="text">
         <string>Check Size</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
